code/datums/spells/construct_spells.dm

```diff
@@ -64,7 +64,11 @@
 	action_icon_state = "summonsoulstone"
 	action_background_icon_state = "bg_demon"
 
+	if(iscultist(user) || iswizard(user))

```

Comment:
might work. i'd ask coderbus, they know everything. here's a link if you don't have an IRC client and stuff already: https://client01.chat.mibbit.com/?url=irc%3A%2F%2Firc.rizon.net%2Fcoderbus